css/interface.css

```diff
@@ -241,7 +246,8 @@ html {
 
 .side-actions {
   position: sticky;
-  top: 10px;  /* gap to the header to avoid touch of elements */
+  top: 10px;
+  /* gap to the header to avoid touch of elements */
```

Comment:
very confusing, this looks like it's applying to the next line of code

Reply:
I agree. **HookyQR.Beautify** does this automatically and I couldn't quite figure out how to turn it off. It seems to insist in putting comments in its own line.

I've moved it to the line above instead of below.

Do you have any suggestions for CSS linting that isn't **HookyQR.Beautify**?

Reply:
What about these:
- http://csslint.net/ (https://github.com/CSSLint/csslint/wiki/Command-line-interface)
- https://stylelint.io/ (ESLint for CSS https://javascript.plainenglish.io/stylelint-the-css-cousin-of-eslint-5701d002da21)